### PR TITLE
changes in Radius and RadiusPacket

### PR DESCRIPTION
**Pull request by wijith7**

### CODE REVIEW

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java [hunk 1]
```diff
@@ -49,7 +50,8 @@ public static RadiusResponse sendRequest(String uid, String pass, byte[] state)
 
         CasqueConfig.loadConfig();
 
-        byte[] buffer = RadiusPacket.formRequestPacket(uid, pass, state);
+        RadiusPacket rpkt = new RadiusPacket();
```
Comment (menakaj) [inline component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java]: Use a meaningful variable name

**Review by menakaj — CHANGES_REQUESTED** [review]

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java (hunk 1 above)
Comment (wijith7) [reply component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java]: Done

**Review by menakaj — APPROVED** [review]

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java [hunk 2]
```diff
@@ -70,12 +72,13 @@ public static RadiusResponse sendRequest(String uid, String pass, byte[] state)
...
-                log.error("Could not get the Datagram responsePacket",ioe);
+                log.error("Could not get the Datagram responsePacket", ioe);
             }
         }
         log.error("Error contacting the CASQUE SNR Server");
         return new RadiusResponse(RadiusResponse.RADIUS_ERROR);
     }
-}
+
+}
```
Comment (Kanapriya) [inline component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java:84]: Add new line here

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java [hunk 3]
```diff
@@ -70,12 +72,13 @@ public static RadiusResponse sendRequest(String uid, String pass, byte[] state)
                 socket.send(packet);
                 socket.setSoTimeout(2500);
                 socket.receive(responsePacket);
-                return RadiusPacket.parsePacket(responsePacket);
+                return radiusPacket.parsePacket(responsePacket);
             } catch (IOException ioe) {
-                log.error("Could not get the Datagram responsePacket",ioe);
+                log.error("Could not get the Datagram responsePacket", ioe);
```
Comment (Kanapriya) [inline component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java:77]: Better to have whitespace when you print an error message, so can identify the error message and exception clearly.
   log.error("Could not get the Datagram responsePacket ", ioe);

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java (hunk 2 above)
Comment (wijith7) [reply component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java:84]: fixed with - https://github.com/wso2-extensions/identity-outbound-auth-casque-snr/pull/8

File: component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java (hunk 3 above)
Comment (wijith7) [reply component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/radius/Radius.java:77]: fixed with https://github.com/wso2-extensions/identity-outbound-auth-casque-snr/pull/8

**Pull request merged**